### PR TITLE
Change "people" to "teams of people" on Pricing page

### PR DESCRIPTION
**Pull request by mattbk**
Changed link to direct to team list.  "Teams of people" sounds friendlier than just "teams" and helps people remember that teams are made of people.

### CODE REVIEW

**Comment by stevepiercy** [comment]
> teams are made of people

My worst fear has come true: teams are soylent green :scream:

**Comment by whit537** [comment]
:tea:

**Comment by whit537** [comment]
> a green wafer

Obviously, I still have to watch this movie.

File: www/about/pricing.spt [hunk 1]
```diff
@@ -83,7 +83,7 @@ subhead = _("Pricing")
 
     <p>Your payments cover hosting and other costs of doing business, and the
     rest goes to the <a
-    href="https://gratipay.com/Gratipay/members/">people</a> who are building
+    href="https://gratipay.com/explore/teams/">teams of people</a> who are building
```
Comment (whit537) [inline www/about/pricing.spt]: This change in wording alters the meaning of the sentence too drastically. The `/Gratipay/members` link is for the people actually on the Gratipay payroll (which is now gone, yes; #3433). `/explore/teams` is a listing of *all* teams on Gratipay, of which the Gratipay team itself is but one. "People" in this sentence refers to the people on the Gratipay team.

**Pull request closed without merging**